### PR TITLE
Update Filipino translations

### PR DESCRIPTION
**Pull request by geeksteph**
## Proposed Changes
1. Update Filipino translations.

## Additional Info.
Any additional info goes here

### CODE REVIEW

**Comment by geeksteph** [comment]
Ready for review.

**Review by noway — APPROVED** [review]
LGTM, very nice 👍🏻

File: src/translations/locales/common_tl-PH.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "hours": "Oras",
     "closed": "Sarado",
     "publicHolidays": "Public Holidays",
```
Comment (gpoliko) [inline src/translations/locales/common_tl-PH.json:64]: Do we have a translation for Public Holidays?

Comment (gpoliko) [reply src/translations/locales/common_tl-PH.json:64]: Sweet! Cheers @geeksteph! 😃

**Pull request merged**